code/upaya/lib/security_question_populator.rb

```diff
@@ -0,0 +1,31 @@
+module SecurityQuestionPopulator
+  def populate_security_questions
+    add_old_questions
+    add_new_questions
+    update_inactive_questions
+  end
+
+  def add_old_questions
+    old_questions = I18n.t('devise.security_questions.question_array')
+
+    old_questions.each_with_index do |old_question, index|
+      SecurityQuestion.find_or_create_by!(question: old_question, old_index: index + 1)
+    end
+  end
+
+  def add_new_questions
+    new_questions = I18n.t('devise.security_questions.new_questions')
+
+    new_questions.each do |question|
+      SecurityQuestion.find_or_create_by!(question: question)
+    end
+  end
+
+  def update_inactive_questions
+    inactive_questions = I18n.t('devise.security_questions.inactive_questions')
+
+    inactive_questions.each do |question|
+      SecurityQuestion.find_by_question(question).update(active: false)
+    end
+  end
+end
```

Comment:
We don't need all this logic here. The purpose of this module in ICAM was to be able to update the security questions while allowing existing users to use old questions that were removed. In our case, we just need the `add_new_questions` method. However, you didn't import the list of questions in the localized file, so we can remove this file entirely until we need to add security questions.